tests/Commands/EsIndexScoresQueueTest.php

```diff
@@ -48,6 +48,7 @@ public function testParameterValidity(array $params, bool $isValid)
 
     /**
      * @dataProvider dataProviderForTestQueueScores
+     * @group RequiresScoreIndexer
```

Comment:
this doesn't need score indexer as all it does is pushing to a different queue (`static::SCHEMA`)